### ISSUE TITLE
Consider phasing out config.json

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#1224]
The `config.json` file in `.securedrop_client` currently only holds the submission key fingerprint, which already exists in `dom0`. To simplify our provisioning logic (and particularly to make it largely cost-free to wipe `.securedrop_client`), it may be desirable to phase out use of this file.

At least for this particular use, one way to derive the fingerprint may be to ask `sd-gpg` to do it for us. The submission key exists on disk in `sd-gpg` as `~/.gnupg/sd-journalist.sec`, so we can ask it to fingerprint it (these days, the way to do that is a mouthful: `gpg --import --import-options show-only --with-fingerprint [filename]` -- perhaps throw in a `--with-colons` for machine-friendly output).

**Comment by creviera** [comment]
I agree this would be optimal. It's also hard to remember the two different property names for fingerprint between dom0 and sd-app.

**Comment by eloquence** [comment]
This was done in #1883.
